### PR TITLE
How to get started with Agoric for Hackathon devs

### PR DESCRIPTION
**Pull request by tyg**
- Rewritten version of Documentation Guide. Note that due to being on a Chromebook, I'm not able to test it locally after VuePress processing.

### CODE REVIEW

File: main/getting-started/intro-hackathon.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
+# Getting Started for the Agoric Hackathon
```
Comment (rowgraus) [inline main/getting-started/intro-hackathon.md:1]: Let's title this Getting Started for HackTheOrb, The Agoric x Chainlink Hackathon!

File: main/getting-started/intro-hackathon.md [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+and what Agoric API commands do, read the following:
+
+- ERTP: The **[ERTP Guide](/ertp/guide/README.md)** is a detailed description of 
+  ERTP concepts, design, components, and commands. The ERTP API is documented
+  [here](/ertp/api/#ertp-api). 
+- Zoe: The **[Zoe Guide](/zoe/guide/README.md)** is a detailed description of 
+  Zoe concepts, design, components, and commands. The Zoe API is documented
+  [here](/zoe/api/#zoe-api).
+  
+## Install and run the Agoric platform
```
Comment (rowgraus) [inline main/getting-started/intro-hackathon.md:55]: This section should be at the top

**Review by rowgraus — COMMENTED** [review]
Made a few comments but otherwise looks great!  Thanks!

File: main/getting-started/intro-hackathon.md [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+- [Second-price auction](/zoe/guide/contracts/second-price-auction.md)
+- [Simple Exchange](/zoe/guide/contracts/simple-exchange.md)
+
+## Advanced or specialized documentation
+
+As you start writing contracts, you should read our
+**[JavaScript Distributed Programming Guide](/distributed-programming.md)**. 
+We've made some Agoric-specific additions at various layers, including concepts, syntax, 
+and additions to the Agoric library. You should know about and understand these before 
+doing significant programming on the Agoric platform.
```
Comment (Chris-Hibbert) [inline main/getting-started/intro-hackathon.md:83]: If they are Agoric-specific additions, they must have been additions to things the developer might already have been familiar with. I suspect you mean javascript, blockchain, and perhaps distributed programming, but this doesn't say any of that.

Here's a start at actually saying that.

```suggestion
**[JavaScript Distributed Programming Guide](/distributed-programming.md)**. 
The Agoric platform allows you to write smart contracts in JavaScript, and run them where they'll be able to communicate with other programs running on Cosmos, Ethereum, or BitCoin. We've made some Agoric-specific additions to JavaScript, and added functionality that makes distributed programming both more secure and more approachable than other platforms. This guide presents concepts, syntax, 
and tools available on our platform. You should know about and understand these before 
doing significant programming on the Agoric platform.
```

File: main/getting-started/intro-hackathon.md [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+To familiarize yourself with working Agoric smart contracts, take a look at our 
+[sample contracts](/zoe/guide/contracts/README.md). They are:
+- [Atomic Swap](/zoe/guide/contracts/atomic-swap.md)
+- [Autoswap](/zoe/guide/contracts/autoswap.md)
+- [Barter Exchange](/zoe/guide/contracts/barter-exchange.md)
+- [Covered Call](/zoe/guide/contracts/covered-call.md)
+- [Multipool Autoswap](/zoe/guide/contracts/multipoolAutoswap.md)
+- [Second-price auction](/zoe/guide/contracts/second-price-auction.md)
+- [Simple Exchange](/zoe/guide/contracts/simple-exchange.md)
+
```
Comment (katelynsills) [inline main/getting-started/intro-hackathon.md:76]: We have many more contracts than this. Can we add them to the list and link to Github if there is not yet adequate documentation?

File: main/getting-started/intro-hackathon.md [hunk 5]
```diff
@@ -0,0 +1,103 @@
...
+  install dependencies, initialize, deploy, and start Agoric projects. It also tells how to use 
+  the Agoric CLI to develop and test multiuser Dapps by deploying and starting a local-chain-multiuser scenario.
+- **[Deploying Smart Contracts](./deploying.md)**: A brief description of the tools and processes 
+  for deploying contracts to the chain and application code to the application server.
+- **[dIBC (Dynamic Inter-Blockchain Communication Protocol) Guide](https://github.com/Agoric/agoric-sdk/blob/master/packages/SwingSet/docs/networking.md)**:
+  Our enhancement to [IBC](https://cosmos.network/ibc) for connecting to services on other blockchains or making
+  services on the Agoric blockchain available to other blockchains. 
+
+Finally, our ultimate documentation is our **[GitHub
+repository](https://github.com/Agoric/)** for the code that defines the Agoric SDK.
```
Comment (katelynsills) [inline main/getting-started/intro-hackathon.md:97]: If the link text says repo, let's actually link to that, not the Agoric organization 
```suggestion
repository](https://github.com/Agoric/agoric-sdk)** for the code that defines the Agoric SDK.
```

**Review by katelynsills — APPROVED** [review]
Some comments, but otherwise looks good!

**Comment by katelynsills** [comment]
I had a thought - it would be really nice on this page to actually display the lines of code to get started, like the instructions here: https://github.com/Agoric/dapp-fungible-faucet#fungible-faucet-dapp Even if the users actually have to install the prerequisites first, as long as we mention and link to those, by listing the instructions we convey how easy it is. Our current documentation is extensive but seems intimidating, and this would counter that

**Pull request merged**